bootstrap.py

```diff
@@ -15,13 +15,22 @@
 from urllib import urlencode
 from ConfigParser import SafeConfigParser
 
-
-HOSTNAME  = socket.getfqdn()
-HEXMAC    = hex(getnode())
-NOHEXMAC  = HEXMAC[2:]
-MAC       = NOHEXMAC.zfill(13)[0:12]
-RELEASE   = platform.linux_distribution()[1]
-API_PORT  = 443
+def get_architecture():
+  # May not be safe for anything apart from 32/64 bit OS
+  is_64bit = sys.maxsize > 2**32
+  if is_64bit:
+     return "x86_64"
+  else:
+     return "x86"
+
+HOSTNAME     = platform.node().split('.')[0]
```

Comment:
please do not use `platform.node()` as it will not always return an FQDN
use `socket.getfqdn()` instead